src/modules/NtpClient.rb

```diff
@@ -812,13 +812,13 @@ def update_cron_settings
       if @synchronize_time
         SCR.Write(
           path(".target.string"),
-          @cron_file,
+          CRON_FILE,
```

Comment:
Just an idea, ... but probably too late: Could we have a comment here that this is created by us and that the user should run YaST to re/configure it?

Reply:
yes, such comment probably makes good sense. This one is already merged, but I can open new one.

Reply:
Please :)